packages/playground-examples/copy/pt/3-7/Fixits/Const to let.ts

```diff
@@ -0,0 +1,12 @@
+//// { compiler: {  }, order: 1 }
+
+// No novo 3.7 a capacidade de converter rapidamente
+// uma variável const para uma let quando o valor
+// for redesignado.
+
+// Você pode tentar isso destacando o erro abaixo
+// e optando por executar a correção rápida.
```

Comment:
```suggestion
// Você pode experimentar isso destacando o erro abaixo
// e optando por executar o _Quick Fix_ (Correção rápida).
```
![image](https://user-images.githubusercontent.com/17657014/90267656-41acea80-de2c-11ea-8cc9-b87f6111fdde.png)